onnx_tf/common/tf_helper.py

```diff
@@ -2,21 +2,21 @@
 import numpy as np
 
 
-def tf_shape(tensor):
-    """
+def tf_shape(tensor, dtype=tf.int64):
```

Comment:
Please provide description for new optional parameter in the comments area for the function. Seems like it has to be a tf dtype, can't be a numpy.dtype.

Reply:
@chinhuang007 Done. May you review again please.